django/conf/global_settings.py

```diff
@@ -70,6 +70,7 @@ def gettext_noop(s):
     ("en", gettext_noop("English")),
     ("en-au", gettext_noop("Australian English")),
     ("en-gb", gettext_noop("British English")),
+    ("en-ca", gettext_noop("Canadian English")),
```

Comment:
Swap with `en-gb` to keep it alphabetized.

Reply:
OK, we don't have a separate translations for `en_CA` so it should be enough to add `formats.py`.